runtime/src/main/java/org/corfudb/runtime/exceptions/CantCloneLayoutException.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime.exceptions;
+
+/**
+ *
+ * An exception that is thrown when the layout clone operation fails.
+ *
+ * @author Maithem
+ */
+public class CantCloneLayoutException extends RuntimeException {
```

Comment:
Can you move this into unrecoverable? We can have the Error vs Exception debate later, but this is something a programmer reasonably can't expect to recover from...

Reply:
extend from unrecoverable, or throw unrecoverable from the clone method?